troposphere/static/js/components/images/MyImageRequestsPage.jsx

```diff
@@ -128,8 +128,8 @@ export default React.createClass({
                 endDateText = moment(request.get("end_date")).format("MMM D, YYYY h:mm:ss a");
             }
 
-            var newMachineId = !!request.get("new_machine") ? request.get("new_machine").uuid : "N/A";
-            var newMachineProvider = !!request.get("new_machine_provider") ? request.get("new_machine_provider").name : "Unknown";
+            var newMachineId = request.get("new_machine") ? request.get("new_machine").uuid : "N/A";
+            var newMachineProvider = request.get("new_machine_provider") ? request.get("new_machine_provider").name : "Unknown";
```

Comment:
Just wanted to comment, that like you said !! is almost never necessary! because !!expr looks at the truthyness of `expr` if you had used `expr` anywhere else expecting a truthy value it would have relied already on the the truthyness of `expr`.